.github/workflows/run-tests.yml

```diff
@@ -48,10 +47,50 @@ jobs:
           go-version: "1.20"
           check-latest: true
 
-      - name: Build dependencies
+      - name: Install Go dependencies
         run: |
-          make deps:modules
-          make deps:test
-      
+          go mod download
+          go install gotest.tools/gotestsum@latest
+          go install github.com/ory/go-acc@latest
+
+      - name: Build Lens dependencies
+        run: |
+          rustup target add wasm32-unknown-unknown
+          cargo build --target wasm32-unknown-unknown --manifest-path "./tests/lenses/rust_wasm32_set_default/Cargo.toml"
+	        cargo build --target wasm32-unknown-unknown --manifest-path "./tests/lenses/rust_wasm32_remove/Cargo.toml"
+	        cargo build --target wasm32-unknown-unknown --manifest-path "./tests/lenses/rust_wasm32_copy/Cargo.toml"
+
       - name: Run the tests
-        run: make test:${{ matrix.tests }}
+        run: gotestsum ./... --format testname -- -race -shuffle=on -timeout 300s
+
+      - name: Generate test coverage
+        run: go-acc ./... --output=coverage.txt --covermode=atomic -- -failfast
```

Comment:
todo/question: This looks like it is running the tests twice, once without code coverage with our test params (race/shuffle/etc), and once with code coverage but without the params?  Suggest just running the tests once

Reply:
I converted coverage to the new go 1.20 aggregate mode and it now only runs tests once.